### DISCUSSION TITLE
[TS3] Change Query IP for monitor

### DISCUSSION THREAD

**Discussion by SunnyMoo** [GameServerManagers/LinuxGSM#3944] [Q&A] [answered]
Hi,
I've got a problem with lgsm monitor: it tries to query my ts3 server on public IP but query is only allowed on private (vpn) IP and I don't want to let it opened on public IP.
How can I force lgsm monitor to use correct IP to check query ?
Thanks !

**Comment by h3o66** [comment, marked as answer]
You can try the following:

Add the line with the proper IP to the config:
`configip="192.168.3.4"`
to the config:
`lgsm/config-lgsm/ts3server/ts3server.cfg`

You can test if that works for now.

EDIT: maybe as well try it with the default `ip` parameter if the `configip` one does not work intended.

**Comment by SunnyMoo** [comment]
Hi h3o66 and thank you for your answer,
configip doesn't work, and... with ip="x.x.x.x" it works !
I can see correct queries into /home/ts3server/log/script/ts3server-script.log
You should add this trick to the wiki.
Thank you very much !

**Reply by h3o66** [reply]
Thanks for the update.

Well, this is a global setting, so only a workaround for this ( if you have a different ip bound for the query interface )
